bsip-0073.md

```diff
@@ -168,34 +185,59 @@ After the hard fork, when processing a settle order after the delay,
 ## API
 
 APIs which return `settle_order_object` need to return the new `fill_asap`
-field.
+field in the `settle_order_object`.
 
-APIs which return combined order book can combine settle orders whose
-`fill_asap` is true with limit orders in the same direction.
+APIs which return a combined order book can combine settle orders whose
+`fill_asap` is true with the limit orders on the same side of the market.
 
 ## CLI
 
-There was a `settle_asset` command in CLI.
+Currently there is a `settle_asset` command in CLI which can be used to
+create force-settlement orders. After the hard fork, we need a command in
+CLI for users to create force-settlement orders with the new `fill_asap`
+option.
 
-Need to add a new command E.G. `settle_asset_ext` in CLI so that users would
-be able to create force-settlement orders with the new `fill_asap` option.
+One option is to add an optional boolean parameter to the parameter list of
+the existing `settle_asset` command, if it doesn't break existing client
+applications which rely on that wallet API. Otherwise, a new command is
+needed, E.G. `settle_asset_ext`.
 
 ## GUI/UX
 
```

Comment:
Perhaps add a note that UI changes are only suggestions and formally not part of the specification.